jaxrs-service/src/main/java/com/quorum/tessera/api/exception/EntityNotFoundExceptionMapper.java

```diff
@@ -9,13 +12,17 @@
 @Provider
 public class EntityNotFoundExceptionMapper implements ExceptionMapper<EntityNotFoundException> {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(EnclaveNotAvailableExceptionMapper.class);
+
     @Override
-    public Response toResponse(EntityNotFoundException e) {
+    public Response toResponse(final EntityNotFoundException ex) {
+        LOGGER.error("Entity not found: {}", ex.getMessage());
+        LOGGER.debug(null, ex);
+
         return Response.status(404)
```

Comment:
Did you miss changing this one to `Response.status(Response.Status.NOT_FOUND)`

Reply:
Yep, updated. Thanks!